dms/views/directory.xml

```diff
@@ -0,0 +1,736 @@
+                        </page>
+                        <page name="mail_alias" string="Email Alias">
+                            <group>
+                                <group>
+                                    <field
+                                        name="alias_contact"
+                                        string="Accept Emails from"
+                                    />
+                                    <field name="alias_process" />
+                                </group>
+                                <group>
+                                    <label for="alias_name" string="Email Alias" />
+                                    <div>
+                                        <field name="alias_name" class="oe_inline" />
+                                        <span class="oe_read_only">@</span>
+                                        <span class="oe_edit_only"> @ </span>
+                                        <field
+                                            name="alias_domain"
+                                            placeholder="mail.catchall.domain"
+                                            class="oe_inline"
+                                            readonly="1"
+                                        />
+                                    </div>
+                                    <field
+                                        name="alias_id"
+                                        readonly="1"
+                                        required="0"
+                                        groups="base.group_no_one"
+                                        string="Alias Setting"
+                                    />
+                                </group>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" />
+                    <field name="activity_ids" />
+                    <field name="message_ids" />
+                </div>
+            </form>
+        </field>
+    </record>
+    <record id="view_dms_directory_manager_form" model="ir.ui.view">
+        <field name="name">dms_directory.form</field>
+        <field name="model">dms.directory</field>
+        <field name="inherit_id" ref="view_dms_directory_form" />
+        <field name="arch" type="xml">
+            <field name="is_root_directory" position="attributes">
+                <attribute name="invisible">0</attribute>
+                <!-- TODO Fix Update Error
+                <attribute name="groups">dms.group_dms_manager</attribute>
+                -->
+            </field>
+            <field name="storage_id" position="attributes">
+                <attribute name="options">{}</attribute>
+                <!-- TODO Fix Update Error
+                <attribute name="groups">dms.group_dms_manager</attribute>
```

Comment:
These should be fixed before merge

Reply:
For the first one, it can be fixed the following way:
```
<field name="is_root_directory" position="after">
    <field name="is_root_directory" groups="dms.group_dms_manager"/>
</field>
```
The original stays invisible, the second one is displayed only for managers so it's the same as before.

For the second one, I don't see any solution that isn't an overkill hack, so unless somebody has a better idea I would just remove it altogether. There isn't a huge usability impact without it.